assets/js/checkboxHandler.js

```diff
@@ -0,0 +1,25 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
I'd probably have called this addResidentDetail or something.. we could introduce a pattern for page specific stuff 
have them in a `/pages/` folder or something